pkgs/os-specific/linux/pam/default.nix

```diff
@@ -1,65 +1,90 @@
-{ stdenv, buildPackages, fetchurl, fetchpatch, flex, cracklib, db4 }:
+{ stdenv, buildPackages, fetchurl, fetchpatch, flex, cracklib, db4, redHatModules ? false, autoreconfHook ? null, yacc ? null, pkgconfig ? null, libxslt ? null, libxml2 ? null, docbook_xsl ? null, docbook_xml_dtd_44 ? null, docbook_xml_dtd_43 ? null, w3m ? null, docbook_xml_dtd_412 ? null }:
```

Comment:
minor, but please order attributes vertically instead of horizontally. It reduces the diff when changes are needed